custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -674,6 +676,10 @@ def check_transform_server(self, target_temp_location=None):
                 payload.update({"y": open(target_location)})
 
             response = requests.post(run.url_server_address + endpoint, files=payload)
+            transformed_values = read_x_data_from_response(response)
+            if self._schema_validator:
```

Comment:
when would the validator not be present?

Reply:
if the run mode isn't fit or push.

Reply:
this is only called for fit. I don't think you need the check.

Reply:
ok